src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -207,4 +208,28 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addTwigFormatOptions(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('date')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('format')->defaultNull()->end()
+                        ->scalarNode('interval_format')->defaultNull()->end()
```

Comment:
but having concrete defaults makes it easier to understand and also serves as an example value. furthermore, setNumberFormat does not support null. so for it the default is overwritten anyway. this is why is should be done for both consistently.